ee/clickhouse/models/event.py

```diff
@@ -105,6 +109,13 @@ def get_elements_hash(self, event):
         return event[6]
 
 
+def _unpad_property(val: str):
+    if val.isdigit() or val == "true" or val == "false":
```

Comment:
Isn't this if clause redundant?

`'true'.strip('"') => 'true'`

Reply:
oh yes it doesn't need the true or false check here!

Reply:
in fact no check is needed. I was confusing myself with the reverse of this function that I had to do